pkg/render/intrusiondetection/dpi/dpi_test.go

```diff
@@ -208,6 +210,12 @@ var _ = Describe("DPI rendering tests", func() {
 		dpiCertSecret, err = certificateManager.GetOrCreateKeyPair(cli, render.DPITLSSecretName, common.OperatorNamespace(), []string{""})
 		Expect(err).NotTo(HaveOccurred())
 
+		snortRules, err := embed.AsConfigMap(
+			applicationlayer.ModSecurityRulesetConfigMapName,
+			common.OperatorNamespace(),
+		)
+		Expect(err).To(BeNil())
```

Comment:
I think this whole lot can just be replaced with.. let me know if this is incorrect 

```suggestion
		snortRules := &corev1.ConfigMap{
			TypeMeta: metav1.TypeMeta{Kind: "ConfigMap", APIVersion: "v1"},
			ObjectMeta: metav1.ObjectMeta{
				Name:      dpi.DeepPacketInspectionSnortRulesCmName,
				Namespace: common.OperatorNamespace(),
			},
		}
```